docs/renater_federation_saml.md

```diff
@@ -0,0 +1,73 @@
+### Renater "Féderation Éducation-Recherche" federation use
+
+Marsha allows to use external Identity Provider from the "Féderation Éducation-Recherche"
+using the [Renater Federation](https://services.renater.fr/federation/introduction/index)
+
+
+#### Marsha configuration
+
+There are few settings to define in order to make marsha compliant with SAMLv2 authentication.
+
+You will need to provide following environment variables:
+- `DJANGO_SOCIAL_AUTH_SAML_FER_SP_ENTITY_ID`:  The Marsha's entity ID, it may only be its
+  metadata URL.
+- `DJANGO_SOCIAL_AUTH_SAML_FER_SP_PUBLIC_CERT`: Marsha's X.509 public certificate.
+- `DJANGO_SOCIAL_AUTH_SAML_FER_SP_PRIVATE_KEY`: Marsha's private key associated with
+  the certificate.
+- `DJANGO_SOCIAL_AUTH_SAML_FER_FEDERATION_SAML_METADATA_URL`: The URL of the
+  Renater FER federation metadata.
+- `DJANGO_SOCIAL_AUTH_SAML_FER_ORG_INFO_NAME`: The organization's name
+- `DJANGO_SOCIAL_AUTH_SAML_FER_ORG_INFO_DISPLAY_NAME`: The organization's display name
+- `DJANGO_SOCIAL_AUTH_SAML_FER_ORG_INFO_URL`: The organization's website
+- `DJANGO_SOCIAL_AUTH_SAML_FER_TECHNICAL_CONTACT_NAME`: The name of the technical
+  team maintaining the SAML configuration.
+- `DJANGO_SOCIAL_AUTH_SAML_FER_TECHNICAL_CONTACT_EMAIL`: The email of the technical team.
+- `DJANGO_SOCIAL_AUTH_SAML_FER_SUPPORT_CONTACT_NAME`: The user support team name.
+- `DJANGO_SOCIAL_AUTH_SAML_FER_SUPPORT_CONTACT_EMAIL`: The user support team email address.
+
+
+#### Security
+
+By default, Marsha is quite strict with SAML required specification:
+ - Marsha signs its authentication requests
+ - Marsha signs its metadata
+ - Marsha expects message to be signed in the authentication response
+ - Marsha expects assertion to be signed in the authentication response
+ - Marsha expects assertion to be encrypted in the authentication response
+ - Marsha rejects deprecated algorithms
+
+These values are not editable by environment variable for now, but they are all stored in the
```

Comment:
Why not allowing to modify them ?

Reply:
We can, as it may raise security issues (if all are disabled) I thought it was better to enforce them. But you are right, I probably should allow deployment configuration to be able to tweak them in case the FER does not accept them all. I will let them enabled by default but allow configuration to disable each one.

Reply:
Ok for now we can keep it like this.